CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+
+```
+pip_interop_enabled: true
+```
+
+## Commit Message Guidelines
+
+The guidelines below are for commit messages. While currently not enforced, eventually we will enforce the guidelines and use them for the [CHANGELOG.md][changelog-uri] generation and we strongly encourage following the guidelines.
+
+```text
+<TYPE>(<SCOPE>): imperative subject in present tense less than 80 characters
+NOTES:
+Add any additional notes in imperative present tense.
+BREAKING CHANGE:
+Describe the previous behavior in contrast to the new behavior and how it might effect consumers.
+```
+
+Allows types are:
+
+* **feat**: represents a new feature
+* **fix**: represents a bug fix
+* **build**: changes to the gitlab pipeline or build scripts
+* **refactor**: refactor code without expecting any behavioral change
+* **test**: updates to test code only
+* **perf**: updates for performance only (should include a benchmark for verification)
+* **docs**: update to documentation only
+* **style**: changes to style only (reformatting, whitespace changes, etc)
+
+The `(<SCOPE>)` argument should only be included if the change is isolated to a single module. This should be the name
+of the module that was modified.
+
+Only include a BREAKING CHANGE footer if necessary. Only include a NOTES body if it adds appropriate value.
+
+The header line **SHOULD** be in all lowercase characters and exclude any special characters other than the semi-colon
+and the parentheses around the scope (if included).
+
+Example commit message:
+
+```text
+docs: add documentation on contributing.
+* initial creation of the contributing documentation.
+NOTES:
+Contirbution follows the usual Github open source contribution mechanism. However, there are some customizations specific to Mentat. The general approach, where to find more information and Mentat specific guidelines are documented here.
+```
+
+## Code review and Merge approval ##
+
+In order to maintain standardization of practices, ensure security standards are being met, and to incorporate third party code as seamlessly as possible, all submitted code will go through a code review and merge approval process.
+
+Code contributors are able to coordinate with the Mentat team at any point during the contribution process. We recommend initiating the discussions as early as possible, to decrease the likelihood of issues around merging or using the contributed code occurring late in the process. 
+
+The type of contribution being made (e.g. typo corrections vs. a new repository), complexity of code change (e.g. adding a new test vs. adding a new function), and the testability of the code (e.g. well-documented and replicable) will factor into the level of interaction needed with the Mentat team.
+
+**Keep in mind this is an open source project manned by volunteers. This means that there is no guarantee on acceptance of contributions or turn around time for responding to any issues, pull requests.**
```

Comment:
Maybe we should omit this line too. Most of the core contributors are actually paid by Abante.

Reply:
Made the changes